starboard/android/apk/app/src/main/java/dev/cobalt/coat/StarboardBridge.java

```diff
@@ -118,6 +118,8 @@ public void run() {
   private static final TimeZone DEFAULT_TIME_ZONE = TimeZone.getTimeZone("America/Los_Angeles");
   private final long timeNanosecondsPerMicrosecond = 1000;
 
+  private final boolean disableBackgroundPlayback = true;
```

Comment:
I think `enableBackgroundPlayback = false` is more clear because it avoids double negative parsing.

Reply:
Done